sample-notificator-app/src/main/groovy/life/qbic/samplenotificator/components/EmailHTMLTemplate.groovy

```diff
@@ -42,6 +42,7 @@ class EmailHTMLTemplate {
         Element projectCode = document.getElementById("project-code")
         projectTitle.empty()
         projectCode.empty()
+        String titleContent = notificationContent.getProjectTitle() ? notificationContent.getProjectTitle() : "n.a."
```

Comment:
Can be simplified by the elvis operator: `?:` [Groovy Docs](https://groovy-lang.org/operators.html#_elvis_operator)

Reply:
Nice catch 👀